### PR TITLE
Validate `ChecklistTemplateItem#expected_duration_days` is >= 0

### PR DESCRIPTION
**Pull request by garyhtou**
### Describe your change in plain English.

Validates that the `expected_duration_days` fields on `ChecklistTemplateItem` is greater than or equal to zero.

### Link to the issue

Addresses this comment: https://github.com/rubyforgood/pet-rescue/pull/111#discussion_r1278410872

### CODE REVIEW

File: app/models/checklist_template_item.rb [hunk 1]
```diff
@@ -21,6 +21,8 @@
 #
 class ChecklistTemplateItem < ApplicationRecord
   belongs_to :checklist_template
-  validates :name, :expected_duration_days, presence: true
   has_many :checklist_assignments, dependent: :destroy
+
+  validates :name, presence: true
+  validates :expected_duration_days, presence: true, numericality: {greater_than_or_equal_to: 0}
```
Comment (edwinthinks) [inline app/models/checklist_template_item.rb:27]: Could you add a test please, there is a nice shoulda matcher we can use to make this happen fast :)

**Review by edwinthinks — CHANGES_REQUESTED** [review]

File: app/models/checklist_template_item.rb (hunk 1 above)
Comment (garyhtou) [reply app/models/checklist_template_item.rb:27]: sorry it took me so long to get around to this!

**Review by nsiwnf — APPROVED** [review]
